Sources/UITableView.swift

```diff
@@ -0,0 +1,83 @@
+import UIKit
+import ReactiveSwift
+import Result
+
+// FIXME: Temporary workaround for the lack of `BindingTarget` without `Lifetime`.
+private let (lifetime, token) = Lifetime.make()
+
+extension NSObject: ReactiveExtensionsProvider {}
+
+extension Reactive where Base: UITableView {
+	public var collection: BindingTarget<(UITableView) -> UITableViewDataSource> {
+		return BindingTarget(lifetime: lifetime) { [weak base] in
```

Comment:
This is my expectation on what eventually happens when the API & implementation are mature. Collection _is_ an essential. That's said I thought we have the repo because we can quickly iterate in this repo from scratch.